cmd/timestamp-server/app/root.go

```diff
@@ -64,10 +64,11 @@ func init() {
 	rootCmd.PersistentFlags().String("address", "127.0.0.1", "Address to bind to")
 	rootCmd.PersistentFlags().String("timestamp-signer", "memory", "Timestamping authority signer. Valid options include: [kms, tink, memory, file]. Memory and file-based signers should only be used for testing")
 	// KMS flags
-	rootCmd.PersistentFlags().String("kms-key-resource", "memory", "KMS key for signing timestamp responses. Valid options include: [gcpkms://resource, azurekms://resource, hashivault://resource, awskms://resource]")
+	rootCmd.PersistentFlags().String("kms-key-resource", "", "KMS key for signing timestamp responses. Valid options include: [gcpkms://resource, azurekms://resource, hashivault://resource, awskms://resource]")
 	// Tink flags
-	rootCmd.PersistentFlags().String("tink-key-resource", "memory", "KMS key for signing timestamp responses for Tink keysets. Valid options include: [gcp-kms://resource, aws-kms://resource]")
-	rootCmd.PersistentFlags().String("tink-keyset-path", "memory", "Path to KMS-encrypted keyset for Tink, decrypted by tink-key-resource")
+	rootCmd.PersistentFlags().String("tink-key-resource", "", "KMS key for signing timestamp responses for Tink keysets. Valid options include: [gcp-kms://resource, aws-kms://resource]")
```

Comment:
In the `kms-key-resource` flag options, `gcpkms://resource` and `awskms://resource` don't have a dash, like `gcp-kms://resource` in this flag option. Can we standardize on both flags either using dashes or not using dashes in the name?

Reply:
That's a fair point. The former came from sigstore/sigstore (and I think was inspired by the industry), the latter is how Tink chose to specify the keys.

One concern is you'd use something like tinkey (https://github.com/google/tink/blob/27b061bb9ed1af1a6f538410bff443290e427e66/docs/TINKEY.md) to generate the keyset, you'd then be specifying a different key prefix for the server, which might be confusing. Thoughts on this? It's easy enough to handle the different prefix in the server if we want to unify the two, but might confuse a regular Tink user.

Reply:
I'm leaning towards using the Tink prefix format for both flags because of the tinkey keyset generation situation you mentioned and because it looks like the other flag format isn't used anywhere else in the code really.

Reply:
The rest of sigstore uses the non-hyphenated one, so we should probably prefer that over Tink. I'm somewhat leaning towards keeping them as-is, because there's other discrepancies besides hyphenation (hcvault vs hashicorp)

A longterm solution could be proposing or updating an existing [AIP](https://google.aip.dev/) to specify how to reference a key by provider, to align sigstore, Tink, and other libraries.

Reply:
Leaving this as is and readdressing in the longterm with an AIP makes sense to me.